src/System.Management.Automation/security/wldpNativeMethods.cs

```diff
@@ -230,7 +230,7 @@ private static SystemEnforcementMode GetAppLockerPolicy(string path, SafeHandle
 
                                 // AppLocker fails when you try to check a policy on a file
                                 // with no content. So create a scratch file and test on that.
-                                string dtAppLockerTestFileContents = AppLockerTestFileContents + DateTime.Now;
+                                string dtAppLockerTestFileContents = AppLockerTestFileContents + DateTime.UtcNow.Ticks;
                                 IO.File.WriteAllText(testPathScript, dtAppLockerTestFileContents);
```

Comment:
It seems the file name does not depend on the DateTime, but rather the file **contents**. So maybe we want to keep human date format

Reply:
The file is created for _automatic_ testing - no humans see and read the file.

Reply:
> no humans see and read the file.

Perhaps it is for debugging purposes.

Actually, if no humans see the file do we require the datetime to be written in any form at all?

Reply:
I don't know. I am only resolving the perf issue. I can only guess it is how AppLocker works. Perhaps @TravisEz13 or @PaulHigin could add more information.

Reply:
@TravisEz13 and @PaulHigin, can you please comment on whether we need `DateTime` to be added to the file content at all?

Reply:
The DateTime was included to add some randomness to the file contents.  I think we still need it, but could change it to some other random value if computing it is quicker than DateTime.

Reply:
@iSazonov Maybe replacing `DateTime.Now.Ticks` with `Environemnt.TickCount`?

Reply:
@daxian-dbw I have no objections. I see `Environment.TickCount64` - maybe use it?

Reply:
That sounds fine to me, go ahead.

Reply:
Done. The PR description has been updated too.